include/rang.hpp

```diff
@@ -111,13 +115,19 @@ namespace rang_implementation {
 		return pLogbuff;
 	}
 
-	inline int getIword()
+#if defined(OS_LINUX) || defined(OS_MAC)
+    inline const char * getTerm() {
```

Comment:
Wow, I'm really embarrassed by this PR o_O . I refactored this down to the static variable inside supportsColor() .